.travis.yml

```diff
@@ -12,6 +12,7 @@ cache:
   directories:
     - vendor
 install:
+  - npm i uber-licence -g
```

Comment:
Would avoiding `-g` be best for travis and just reference by path directly `./node_modules/.bin/...` be more worthwhile?